test/pop/storage/save_load_tree_test.cpp

```diff
@@ -57,6 +57,52 @@ TEST_F(SaveLoadTreeTest, ValidTree) {
   assertTreesEqual();
 }
 
+TEST_F(SaveLoadTreeTest, ReloadWithoutDuplicates_NoUnconnectedBlockCheck) {
```

Comment:
Can you describe this test case more clear. What situation we are trying to check ?

Reply:
Some am I right that second block which will contain duplicated payloads actually marked as invalid in the tree and this block can be stored in the tree even marked as invalid and such test checks that load() function will not fail and correctly load such invalid block ?